docs/dev_guide/ceph_s3_tests_guide.md

```diff
@@ -0,0 +1,143 @@
+# Ceph S3 Tests Guide
+
+## General Settings For Ceph Tests
+
+### 1) Before Building The Images (Noobaa-Core)
+We will use minikube to run the tests. It is recommended to build all images on the minikube docker daemon.  
+To configure your docker client to use minikube's docker run the following before running `make`:
+```bash
+eval $(minikube docker-env)
+```
+
+Note: For users with OS Mac with chip M1: you need to remove `mongoDB` installation in the test docker file `Tests.Dockerfile`, since there are no ARM packages available for it, in any case it is not used in this context.
+```diff
+ RUN dnf group install -y -q "Development Tools" && \
+     dnf install -y -q --nogpgcheck vim \
+-    mongodb-org-3.6.3 \
+-    mongodb-org-server-3.6.3 \
+-    mongodb-org-shell-3.6.3 \
+-    mongodb-org-mongos-3.6.3 \
+-    mongodb-org-tools-3.6.3 \
+     # commenting out as we are not running ceph test (system_test_list) anywhere
+     which python3-virtualenv python36-devel libevent-devel libffi-devel libxml2-devel libxslt-devel zlib-devel \
+     git && \
+```
+
+### 2) Build Operator (Noobaa-Operator)
+We want to build the CLI and the operator image.
+It's easiest to just build `all`, run:
+```bash
+. ./devenv.sh
+make all
+```
+This will build the following
+* noobaa-operator image with the tha tag `noobaa/noobaa-operator:<major.minor.patch>` (for example: `noobaa/noobaa-operator:5.13.0`). this tag is used by default when installing with the CLI.
+* noobaa CLI. The `devenv.sh` script is setting an alias `nb` to run the local build of the CLI.
+
+## 3) Build Core And Tester Images (Noobaa-Core)
```

Comment:
```suggestion
### 3) Build Core And Tester Images (Noobaa-Core)
```
Something looks wrong when you view the file - I think there are some wrong hash levels in the files - please fix